smart-contract/README.md

```diff
@@ -673,20 +727,19 @@ provenanced q bank balances tp145r6nt64rw2rr58r80chp70ejdyqenszpg4d47 -t -o json
 }
 ```
 
-Now, `bank2` can deliver the cash/fiat to `user2` (off chain process). In addition, `bank2` can
-request the debt from `bank1` be paid. They can also sit on the reserve tokens and swap them for
+Now, `bank2` can burn the reserve tokens held. They can also sit on the reserve tokens and swap them for
 `usdf.local` when another user provides cash/fiat.
 
 ## Swap
 
 Let's say `user2` now wants $25 worth of tokens back from `bank2`. The bank can then swap the
-`bank1.coin` they hold back out for `usdf.local`, minting and withdrawing the tokens directly to
+`bank2.coin` they hold back out for `usdf.local`, minting and withdrawing the tokens directly to
 `user2`.
 
 ```bash
 provenanced tx wasm execute \
     tp14hj2tavq8fpesdwxxcu44rty3hh90vhuz3ljwv \
-    '{"swap":{"amount":"2500","denom":"bank1.coin","address":"tp1m4arun5y9jcwkatq2ey9wuftanm5ptzsg4ppfs"}}' \
+    '{"swap":{"amount":"2500","denom":"bank2.coin","address":"tp1m4arun5y9jcwkatq2ey9wuftanm5ptzsg4ppfs"}}' \
```

Comment:
In the result here, https://github.com/provenance-io/digital-currency-consortium/pull/67/files?diff=unified&w=1#diff-9809a2862614e448a144a2e98cd6fd1b06244750b6e8f465700919e46831b16aR753 the coins that user2 have aren't newly minted, right? They are just coming from what bank2 already held?

Reply:
sorry what is that link supposed to be pointing to?

bank2 has some bank2.coin so can use that to:
* move bank2.coin back to marker
* mint usdf.local for equivalent amount
* withdraw usdf.local to specified address

Reply:
Was supposed to link to the new line 753 (the result of this tx). It seems kind of misleading, sounds as if the swap resulted in `usdf.local` being minted

Reply:
yeah swap is really minting `usdf.local` from a bank coin